Whitey Dark theme

gilbertohasnofb opened the pull request.

gilbertohasnofb committed:
Whitey Dark theme

abnerlee commented:
Could you add this as a fork following https://github.com/typora/typora-theme-gallery/pull/74/files? And change from `category: theme` to `category: fork`?

gilbertohasnofb committed:
adding Whitey theme page

gilbertohasnofb committed:
mentioning whitey dark in Whitey.md

gilbertohasnofb commented:
@abnerlee as suggested in https://github.com/typora/typora-theme-gallery/pull/165#issuecomment-724063915 I created a new `.md` file for the built-in Whitey theme, moved my fork to the fork folder, added `fork` to the YAML header, and added a mention to it in the new `Whitey.md` file. Should be ready to be merged.
